src/mesh/FoamMesh.C

```diff
@@ -13,13 +19,31 @@ registerMooseObject("hippoApp", FoamMesh);
 
 namespace
 {
+static std::map<int, int> subdomain_id_map{
+    {libMesh::TRI3, 1}, {libMesh::QUAD4, 2}, {libMesh::C0POLYGON, 3}};
 Foam::fvMesh
 read_polymesh(const Foam::Time & run_time)
 {
   Foam::IOobject mesh_header(
       Foam::fvMesh::defaultRegion, run_time.name(), run_time, Foam::IOobject::MUST_READ);
   return Foam::fvMesh(mesh_header);
 }
+
+std::unique_ptr<Elem>
+getElement(const Hippo::FoamFace & face)
```

Comment:
I would usually interpret `get` to mean retrieving an existing element. Would `makeNewElement` (or similar) be a slightly better name here?

Reply:
Only `face.size()` is used here. This function doesn't really require a `FoamFace` object to run, we should probably just pass a `std::size_t`.